pandas/tests/frame/test_apply.py

```diff
@@ -1404,3 +1404,101 @@ def test_consistency_of_aggregates_of_columns_with_missing_values(self, df, meth
         tm.assert_series_equal(
             none_in_first_column_result, none_in_second_column_result
         )
+
+
```

Comment:
can you create a new test file for these, maybe test_apply_relabling.py (alt we can make a pandas/tests/frame/apply/..... subdir)

Reply:
ok with *moving* existing functions later, but pls create a new file / subdir now.